### DISCUSSION TITLE
Update Verification Email Template

### DISCUSSION THREAD

**Discussion by YoumnaJ** [OrchardCMS/OrchardCore#15059] [Q&A]
Hi , I need to know please how to over write or Update the Orchard Core CMS Verification email templates with another template

**Comment by Piedone** [comment]
You can put a file called `TemplateUserConfirmEmail.cshtml` into your module/theme, perhaps by starting from [the built-in one](https://github.com/OrchardCMS/OrchardCore/blob/16ade0f2e387bc3d6c83f7d66b8e3489ca9e057b/src/OrchardCore.Modules/OrchardCore.Users/Views/TemplateUserConfirmEmail.cshtml), and customize that.

**Reply by YoumnaJ** [reply]
thanks a lot for your help but how to redirect sending confirmation email from default one to the customized one created

**Reply by Piedone** [reply]
You'll need to create a module first, put the cshtml there, then enable the module to get it applied. Check out https://orcharddojo.net/orchard-training/dojo-course-3-the-full-orchard-core-tutorial for a tutorial on how all of these work.

**Reply by YoumnaJ** [reply]
thank you I created the module and I put the .cshtml but I need to know  how to redirect to the new TemplateUserConfirmEmail.cshtml from startup.cs

**Reply by Piedone** [reply]
You don't need to redirect anything. If you enable the module, the template will get applied and override the built-in one. However, do add a dependency on `"OrchardCore.Users"` from your module's manifest.
